azurerm/internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -310,6 +310,41 @@ func resourceCosmosDbAccount() *schema.Resource {
 				},
 			},
 
+			"backup_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+							Default:  string(documentdb.TypePeriodic),
+							ValidateFunc: validation.StringInSlice([]string{
+								string(documentdb.TypeContinuous),
+								string(documentdb.TypePeriodic),
+							}, false),
+						},
+
+						"interval_in_minutes": {
+							Type:         schema.TypeInt,
+							Optional:     true,
+							Computed:     true,
```

Comment:
Persumably the `Computed` here is not needed?

Reply:
If the user specifies the `type` `periodic`, the `interval_in_minutes` will have a default value?

Reply:
I'll add a validation in test.